hearthstone/simulator/core/hero_pool.py

```diff
@@ -320,9 +319,23 @@ class Malygos(Hero):
     base_power_cost = 0
     power_target_location = [CardLocation.BOARD,
                              CardLocation.STORE]  # TODO: are there other hero powers with multiple target locations?
+    multiple_power_uses_per_turn = True
```

Comment:
Maybe instead of special coding the counting for Malygos, we replace multiple_power_uses_per_turn with an optional integer which is the cap (defaulting to 1?)

Reply:
We also have an attribute `hero_power_used`. Would it make sense to replace that as well?

Reply:
yeah definitely!